docs/modules/plugins/pages/plugin-web-app-to-rest-api.adoc

```diff
@@ -110,17 +110,124 @@ Examples:
 |Default
 |Description
 
-|`bdd.transformer.headless.seed-relative-urls`
+4+^.^|_General_
+
+|`transformer.headless.seed-relative-urls`
 |Comma-separated list of values
 |
 |list of relative URLs, a seed URL is a URL that is fetched by the crawler to extract new URLs in it and follow them for crawling
 
-|`bdd.transformer.headless.filter-redirects`
+|`transformer.headless.filter-redirects`
 |`true` / `false`
 |`false`
 |defines whether urls that has redirect to the one that has already been included in the table are excluded from the table
+
+|`transformer.headless.socket-timeout`
+|`integer`
+|`40000`
+|socket timeout in milliseconds
+
+|`transformer.headless.connection-timeout`
+|`integer`
+|`30000`
+|connection timeout in milliseconds
+
+|`transformer.headless.max-download-size`
+|`integer`
+|`1048576`
+|max allowed size of a page in bytes. Pages larger than this size will not be fetched
+
+|`transformer.headless.max-connections-per-host`
+|`integer`
+|`100`
+|maximum connections per host
+
+|`transformer.headless.max-total-connections`
+|`integer`
+|`100`
+|maximum total connections
+
+|`transformer.headless.follow-redirects`
+|`true` / `false`
+|`true`
+|whether to follow redirects
+
+|`transformer.headless.max-depth-of-crawling`
+|`integer`
+|`-1`
+|maximum depth of crawling, for unlimited depth this parameter should be set to -1
+
+|`transformer.headless.max-pages-to-fetch`
+|`integer`
+|`-1`
+|number of pages to fetch, for unlimited number of pages this parameter should be set to -1
+
+|`transformer.headless.politeness-delay`
+|`integer`
+|`0`
+|politeness delay in milliseconds between sending two requests to the same host
+
+|`transformer.headless.max-outgoing-links-to-follow`
+|`integer`
+|`5000`
+|max number of outgoing links which are processed from a page
+
+|`transformer.headless.respect-no-follow`
+|`true` / `false`
+|`false`
+|whether to honor links with https://en.wikipedia.org/wiki/Nofollow[nofollow flag]
+
+|`transformer.headless.respect-no-index`
+|`true` / `false`
+|`false`
+|whether to honor links with https://en.wikipedia.org/wiki/Noindex[noindex flag]
+
+|`transformer.headless.user-agent-string`
+|`string`
+|`crawler4j (https://github.com/rzo1/crawler4j/)`
+|https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent[User agent]
+
+|`transformer.headless.cookie-policy`
+|`ignore`, `standard`, `relaxed`
+|`no default value`
+|Cookie policy as defined per https://hc.apache.org/httpcomponents-client-4.5.x/current/tutorial/html/statemgmt.html#d5e515[cookie specification]
```

Comment:
https://github.com/vividus-framework/vividus/pull/2644/files#r846201252 and here is the upper case